application/src/main/java/uk/nhs/hee/tis/revalidation/service/DoctorsForDBService.java

```diff
@@ -91,16 +88,12 @@ public TraineeSummaryDto getAllTraineeDoctorDetails(final TraineeRequestDto requ
       final List<String> hiddenGmcIds) {
     final var paginatedDoctors = getSortedAndFilteredDoctorsByPageNumber(requestDTO, hiddenGmcIds);
     final var doctorsList = paginatedDoctors.get().collect(toList());
-    final var traineeDoctors = doctorsList.stream().map(d ->
-        recommendationViewMapper.toTraineeInfoDto(d)).collect(toList());
-
-    return TraineeSummaryDto.builder()
-        .traineeInfo(traineeDoctors)
-        .countTotal(getCountAll())
-        .countUnderNotice(getCountUnderNotice())
-        .totalPages(paginatedDoctors.getTotalPages())
-        .totalResults(paginatedDoctors.getTotalElements())
-        .build();
+    final var traineeDoctors = doctorsList.stream()
+        .map(recommendationViewMapper::toTraineeInfoDto).collect(toList());
+
+    return TraineeSummaryDto.builder().traineeInfo(traineeDoctors).countTotal(getCountAll())
```

Comment:
Might just be me but I find this less readable

Reply:
Changed this back